plugin_sge.py

```diff
@@ -144,6 +145,11 @@ def convert_qstat_to_yaml(self, orig_file, out_file, write_method):
         prefix += '_'
         suffix = '.' + suffix
         SGEStatMaker.fd, SGEStatMaker.temp_filepath = get_new_temp_file(prefix=prefix, suffix=suffix, config=self.config)
+
+        if options.TAR >= 1:
```

Comment:
.SAMPLE? (this gives you infinite freedom to make the exact format, such as tarball, a mere tuneable option)